package.json

```diff
@@ -9,8 +9,14 @@
     "example": "examples",
     "test": "test"
   },
+  "bin": {
+    "one-line-logger": "./bin.js"
```

Comment:
should i use one-line-logger name here or something else? @fastify/one-time-logger is not a valid bin name

Reply:
I think simplicity and fast to type thing is better:
- `1l`
- `1line`